nomad-{{cookiecutter.plugin_name}}/py_sources/tests/schema_packages/test_schema.py

```diff
@@ -4,7 +4,7 @@
 
 
 def test_schema():
-    test_file = os.path.join(os.path.dirname(__file__), 'data', 'test.archive.yaml')
+    test_file = os.path.join('tests', 'data', 'test.archive.yaml')
```

Comment:
The old version might actually be better, since it worked no matter where the `pytest` command was run from. This new version assumes that the tests folder is in the cwd...?

Why was the old code changed? Maybe it did not quite work as you need to use the parent directory of `__file__`?

Reply:
The new version works from the parent directory, so the top level directory. 

The previous version included the name of the repo twice in the path: see failing test [here](https://github.com/hampusnasstrom/nomad-sintering-plugin-test/actions/runs/9050375910/job/24865647308)